go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/bufbuild/buf
 
-go 1.20
```

Comment:
I'll look into what is causing the upgrade.

Reply:
`github.com/Microsoft/go-winio@v0.6.2 ` declares go1.21 https://github.com/microsoft/go-winio/pull/315